src/material/datepicker/month-view.ts

```diff
@@ -398,6 +415,15 @@ export class MatMonthView<D> implements AfterContentInit, OnChanges, OnDestroy {
     }
   }
 
+  /** Used for turning a cell number into a date. */
```

Comment:
When writing method descriptions, try to phrase it as a description of what the method does rather than talking about when/how the function is called. Those types of comments are still useful, but should be secondary to the _purpose_ of the function

Reply:
done ✔️